### PR TITLE
Feature/pe sp service outage card

### PR DESCRIPTION
**Pull request by sirfabio80**
My new pull request!

### CODE REVIEW

File: src/pe-sp-service-outage-card/config.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
   "common-name": "Sp Service Outage Card",
   "compatibility": [
     "Jakarta",
     "Kingston",
-    "London"
+    "London",
+    "Madrid"
   ],
   "contributors": [
     "mrfabiop80"
```
Comment (asc2683) [inline src/pe-sp-service-outage-card/config.json]: 💯

File: src/pe-sp-service-outage-card/widget/pe-sp-service-outage-card.u-client-script.js [hunk 2]
```diff
@@ -1,3 +1,89 @@
...
+  }
+
+  c.createIncident = function (outage) {
+    c.server.get({
+      action: 'create',
+      sys_id: c.data.sys_id,
+      outage_sys_id: outage.sys_id
+    }).then(function (response) {
+      c.data = response.data;
+    });
```
Comment (asc2683) [inline src/pe-sp-service-outage-card/widget/pe-sp-service-outage-card.u-client-script.js]: - [ ] add if error block with notification to user of error message

File: src/pe-sp-service-outage-card/widget/pe-sp-service-outage-card.u-css.scss [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  box-shadow: none;
+}
+
+.btn-link-danger {
+  color: $brand-danger !important;
+}
+
+.btn-link-success {
+  color: $brand-success !important;
+}
```
Comment (asc2683) [inline src/pe-sp-service-outage-card/widget/pe-sp-service-outage-card.u-css.scss]: use !important sparingly. use css specificity.
https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

File: src/pe-sp-service-outage-card/widget/pe-sp-service-outage-card.u-css.scss [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+.noselect {
+  -webkit-touch-callout: none;
+  /* iOS Safari */
+  -webkit-user-select: none;
+  /* Safari */
+  -khtml-user-select: none;
+  /* Konqueror HTML */
+  -moz-user-select: none;
```
Comment (asc2683) [inline src/pe-sp-service-outage-card/widget/pe-sp-service-outage-card.u-css.scss]: i don't think we need to support Konqueror HTML
- [ ] please remove/delete

File: src/pe-sp-service-outage-card/widget/pe-sp-service-outage-card.u-css.scss [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  /* Firefox */
+  -ms-user-select: none;
+  /* Internet Explorer/Edge */
+  user-select: none;
+  /* Non-prefixed version, currently
+                                  supported by Chrome and Opera */
+}
+
+
+@media (min-width: 768px) {}
```
Comment (asc2683) [inline src/pe-sp-service-outage-card/widget/pe-sp-service-outage-card.u-css.scss]: - [ ] unused, please delete

File: src/pe-sp-service-outage-card/widget/pe-sp-service-outage-card.u-css.scss (hunk 3 above)
Comment (sirfabio80) [reply src/pe-sp-service-outage-card/widget/pe-sp-service-outage-card.u-css.scss]: Thanks! Totally agree, I was in a hurry 🙈

File: src/pe-sp-service-outage-card/config.json (hunk 1 above)
Comment (sirfabio80) [reply src/pe-sp-service-outage-card/config.json]: Thanks!

**Pull request merged**